base/random.jl

```diff
@@ -1287,6 +1284,10 @@ let Floats = Union{Float16,Float32,Float64}
             $randfun(                             dims::Integer...               )           = $randfun(GLOBAL_RNG, Float64, dims...)
         end
     end
+	
+	# complex randn
+	Base.@irrational SQRT_HALF 0.7071067811865475244008  sqrt(big(0.5))
+	randn(rng::AbstractRNG, ::Type{Complex{T}}) where {T <: Floats} = Complex{T}(SQRT_HALF * randn(rng, T), SQRT_HALF*randn(rng, T))
```

Comment:
I'm not sure if we want `T<:Floats` or `T<:AbstractFloat`. Is someone defines `randn(::CustomRNG, ::BigFloat)`, she won't benefit from your complex `randn` definition...